### PR TITLE
add: routes to the app

### LINKED ISSUE #47: research react -router

### PR DESCRIPTION
**Pull request by omriza5**
close #47 
*)added initial routes
*)remove the state from App component
*)edit CSS files in some component
*)remove Joi package for validation

### CODE REVIEW

**Review by ebraheemabbas — DISMISSED** [review]

**Comment by jamiecoe** [comment]
You need to add which issues this PR closes

File: the-experts-fe/src/App.js [hunk 1]
```diff
@@ -1,71 +1,21 @@
...
-          description1="we Are always here for you "
-          description2="if you want to ask or notify us about something just call "
-        />
-        <WhyUs
-          imageSrc="https://cdn3.iconfinder.com/data/icons/miscellaneous-16-solid/128/credible_reliable_Dependable_authentic_credibility_trust-512.png"
-          description1="best Value for the money"
-          description2="We will grantee that you will get the best service"
-        />
+      <NavBar firstParm="Login" secondParam="Register" />
+      <div>
```
Comment (jamiecoe) [inline the-experts-fe/src/App.js]: You don't need a `<div>` around `<Switch>` component

File: the-experts-fe/src/App.js [hunk 2]
```diff
@@ -1,71 +1,21 @@
...
+        <Switch className="App">
+          <Route path="/register" component={RegisterationForm}></Route>
+          <Route exact path="/" component={Home}></Route>
+        </Switch>
       </div>
+
       </div>
```
Comment (jamiecoe) [inline the-experts-fe/src/App.js]: Looks like you've left an extra closing `</div>` here. If it's unnecessary you should remove it

File: the-experts-fe/src/components/registerationForm/registerationForm.jsx [hunk 3]
```diff
@@ -1,49 +1,20 @@
...
-  };
 
   const handleSubmit = e => {
     e.preventDefault();
-    const errors = validate();
-
-    setErrors({ ...errors });
-    if (errors) return;
 
     // make a post request with the contractor object to ebraheem
```
Comment (jamiecoe) [inline the-experts-fe/src/components/registerationForm/registerationForm.jsx:10]: Do you still need this comment? The code explains what you are doing

**Review by jamiecoe — CHANGES_REQUESTED** [review]

**Review by jamiecoe — APPROVED** [review]

**Review by HosenAgbaria — APPROVED** [review]

**Review by ebraheemabbas — APPROVED** [review]

**Pull request merged**